### DISCUSSION TITLE
Will it support app application testing in the future

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2556] [Q&A] [answered]
Will it support app application testing in the future

**Comment by mdmintz** [comment, marked as answer]
You can test any website app right now.

Did you mean mobile app testing? There's a mobile device emulator: [help_docs/mobile_testing.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/mobile_testing.md).
Beyond that, if you're looking for native mobile app testing, try Appium.
